bookwyrm/views/feed.py

```diff
@@ -66,6 +68,7 @@ def get(self, request, tab, filters_applied=False):
                 "path": f"/{tab['key']}",
                 "annual_summary_year": get_annual_summary_year(),
                 "has_tour": True,
+                "has_read_throughs": True if len(readthroughs) else False,
```

Comment:
You could write this a little more simply as:

```py
"has_read_throughs": len(readthroughs) > 0,
``` 
which will do the same thing as it's implicitly returned. This should fix the pylint error.